packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -41,21 +46,27 @@ class UsaBanner extends React.PureComponent {
             <div className="ds-c-usa-banner__guidance">
               <DotGovIcon className="ds-c-usa-banner__icon" />
               <p className="ds-c-usa-banner__media-body">
-                <strong>Official websites use .gov</strong>
-                <br />A <strong>.gov</strong> website belongs to an official government organization
-                in the United States.
+                <strong>{Translation.usaBanner.domainHeaderText}</strong>
```

Comment:
This is one of the nice things about i18next
https://react.i18next.com/guides/the-drawbacks-of-other-i18n-solutions#can-you-translate-combined-jsx-nodes-in-one-sentence

Reply:
Will add another ticket to add translation framework.